pkgs/misc/uboot/default.nix

```diff
@@ -366,6 +391,103 @@ in {
     filesToInstall = ["u-boot-sunxi-with-spl.bin"];
   };
 
+  ubootOrangePiR1PlusLts = let
+    rkbin = fetchFromGitHub {
+      owner = "armbian";
+      repo = "rkbin";
+      rev = "5d409529dbbc12959111787e77c349b3e832bc52";
+      sha256 = "09r4dzxsbs3pff4sh70qnyp30s3rc7pkc46v1m3152s7jqjasp31";
+    };
+  in buildUBoot {
+    version = "2022.07";
+    src = fetchFromGitHub {
+      owner = "u-boot";
+      repo = "u-boot";
+      rev = "v2022.07";
+      sha256 = "1EONRmYLsD0uxo+kpE6mLIYkYMU09Yt0EvSbHhj5prw=";
+    };
```

Comment:
We do not actually support anything else than the main version from `builtUBoot` directly in Nixpkgs.

Making sure the build works with arbitrary older revisions is an actual issue.